source/unified-test-format/unified-test-format.rst

```diff
@@ -462,6 +478,49 @@ The structure of this object is as follows:
 
     Test files SHOULD NOT use this option unless one or more command monitoring
     events are specified in `observeEvents <entity_client_observeEvents_>`_.
+    
+  - ``storeEventsAsEntities``: Optional map of entity names to an array of
+    event names.
+    
+    For each entity name, the test runner MUST create the respective entity
+    with a type of "event list", as described in `Supported Entity Types`_.
+    If the entity already exists (such as from a previous
```

Comment:
> If the entity already exists (such as from a previous ``storeEventsAsEntities`` declaration from another client),

This is already enforced by the spec. It is stated in the `Entity Map` section:
```
Test runners MUST raise an error if an attempt is made to store an entity with a name that already exists in the map
```

I think re-stating it here does not add any value.

Reply:
To me the difference between event lists and, say, client entities is that the client entities are fully constructed when placed in the entity map but event lists are placeholders and are populated later. Given this distinction I think it's helpful to remind implementers that each operation requires its own entity.

Reply:
SGTM. Resolving.